beetmoverscript/script.py

```diff
@@ -318,6 +321,12 @@ def list_bucket_objects(context, s3_resource, prefix):
                     upload_to_s3(context=context, s3_key=destination, path=source)
                 )
             )
+
+            if is_partner_public_task(context):
+                context.checksums[full_path_artifact] = {
+                    algo: get_hash(source, algo) for algo in context.config['checksums_digests']
+                }
+
```

Comment:
I think you also need something like `context.checksums[artifact_pretty_name]['size'] = get_size(source)` too like [here](https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/script.py#L293) as it's being used later on when checksums file is being generated [here](https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/task.py#L84)

Reply:
Thanks for catching that.